### ISSUE TITLE
Mapping commands with arguments?

### ISSUE THREAD

**Issue by varmanishant** [NeoVintageous/NeoVintageous#312]
How do I map a key to open the files just like <kbd>ctrl+p</kbd>?

Here is the definition: `{ "keys": ["ctrl+p"], "command": "show_overlay",  "args": {"overlay": "goto", "show_files": true}}`

I can do  `nnoremap <leader>s :ToggleSideBar<CR>` which performs an _action_ but without _arguments_.

**Comment by gerardroche** [comment]
Currently custom commands can't accept arguments, but you can create a custom command. 

Your custom command will be this:

```
import sublime_plugin

class GotoFilesCommand(sublime_plugin.WindowCommand):
    def run(self):
        self.window.run_command('show_overlay', {
            'overlay': 'goto',
            'show_files": True
        })
```

Then create a mapping e.g.:

```
nnoremap <leader>f :GotoFiles<CR>
```

See http://blog.gerardroche.com/2017/11/01/neovintageous-1.4.0-released/ for more details.

Also, see http://blog.gerardroche.com/2018/02/21/neovintageous-1.5.0-released/ for information on using the `:NeovintageousToggleSideBar` command available since 1.5.0.

**Comment by varmanishant** [comment]
That worked just fine. It leads to a lot of possibilities for us :-)
